docs/tools/buildkit.md

```diff
@@ -123,38 +110,81 @@ If you want to ensure that the buildkit CLI tools are always available, then:
-Apache/MySQL/PHP systems, and you may need to try them a few times.
+Buildkit provides a tool called `amp` which [civibuild](/tools/civibuild.md) uses when it needs to set up a new site. Before you can use `civibuild`, need to configure `amp` by telling it a bit about your system (e.g. what webserver you're using). 
 
-Configure `amp` with details of your Apache/MySQL environment.  Pay close
-attention to the instructions.  They may involve adding a line to your
-Apache configuration file.
+#### Interactive config
 
 ```
 $ amp config
 ```
 
-Test that `amp` has full and correct information about Apache/MySQL.
+!!! tip "tips"
+    * Run this as a non-`root` user who has `sudo` permission. This will ensure that new files are owned by a regular user, and (if necessary) it enables `civibuild` to restart Apache or edit `/etc/hosts`.
+    * Pay close attention to any instructions given in the output of this command.  They may involve adding a line to your Apache configuration file.
+    * To check which version of apache you have, run `apachectl -v`
+
+#### Testing amp's configuration {:#amp-test} 
+
+Test that `amp` is correctly configured.
 
 ```
 $ amp test
 ```
 
-!!! note
-    You may need to alternately restart httpd, re-run `amp config`, and/or re-run `amp test` a few times.
+!!! failure "Troubleshooting errors from `amp test`"
+    
+    * Try manually adding settings to your sebserver, as described below.
+    * Re-run `amp config`.
 
-Create a new build using Drupal and the CiviCRM `master` branch.
-The command will print out URLs and credentials for accessing the website.
+#### Manually adding settings to your webserver {:#amp-webserver}
 
-```
-$ civibuild create dmaster --url http://dmaster.localhost --admin-pass s3cr3t
-```
+During one of the steps in the `amp config` process, `amp` will *attempt to* alter the system-wide settings for your local webserver (apache or nginx). On some platforms, `amp` is not able to perform the necessary configuration and you must do it manually by following these steps:
+
+1. Identify the location of your `amp` installation. It is probably a `.amp` folder within your home directory. Make sure to *use the full path* to this directory in the settings below. We will use `<amp-installation>` henceforth to refer to the full path of this directory. 
+
+1. Identify your webserver. (If using Apache, use `apachectl -v` to see which version you have.)
+
+    * For Apache 2.2: 
+    
+        Create a new file `/etc/apache2/conf.d/buildkit.conf` with the following contents:
+    
+        ```
+        Include <amp-installation>/apache.d/*conf
+        ```
+
+    * For Apache 2.4: 
+    
+        Create a new file `/etc/apache2/conf.d/buildkit.conf` with the following contents:
+    
+        ```
+        IncludeOptional <amp-installation>/apache.d/*conf
+        ```
+
+    * For nginx:
+    
+        Create a new file `/etc/nginx/conf.d/buildkit.conf` with the following contents:
```

Comment:
I see. Do you think it's okay to merge this as-is and assume that people will figure it out? Or do you have something better to propose here?